samples/SkiaJsSample/src/jsWasmMain/kotlin/org/jetbrains/skiko/sample/js/App.jsWasm.kt

```diff
@@ -35,4 +33,9 @@ internal fun runApp() {
         skiaLayer.attachTo(canvas)
         skiaLayer.needRedraw()
     }
-}
\ No newline at end of file
+}
+
+
+// Note: despite the actual implementation looks identical, it can't be common - there is no W3C common API
+// return type is Any because k/js and k/wasm don't have a common type for HTML elements
+expect fun findHTMLCanvasElementById(id: String): Any
```

Comment:
two points
* Nowhere in this code we are relying on this returning actually HTMLCanvas - whatever check happen, if happen at all, happens no here
* Forcing non-nullability in one place is better that in many places

So my suggestion would be to just rename this to something more generic, like just findElementById, make that function return Any? 
```
actual fun findElementById(id: String): Any? = document.getElementById(id)
```
and then just `val canvas = findElementById("c$index")!!`

Reply:
fixed